### PR TITLE
Add health edit

### PR DESCRIPTION
**Pull request by Luzagar**

### CODE REVIEW

File: modules/menus/menu_amounts/src/amounts_menu.cpp [hunk 1]
```diff
@@ -41,6 +43,16 @@ void AmountsMenu::draw() {
     }
 
     switch (cursor.y) {
+    case HEALTH_INDEX:
+    Cursor::moveList(l_healthNum);
+     if (l_healthNum < 1) {
```
Comment (kipcode66) [inline modules/menus/menu_amounts/src/amounts_menu.cpp]: We might want to set the health to 0 to kill link.

File: modules/menus/menu_amounts/include/amounts_menu.h [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #include "menu.h"
 
 struct AmountsData {
+    uint16_t l_healthNum;
```
Comment (kipcode66) [inline modules/menus/menu_amounts/include/amounts_menu.h:4]: Why not also add a setting for changing the Max Health too while we're at it?

File: modules/menus/menu_amounts/src/amounts_menu.cpp [hunk 3]
```diff
@@ -41,6 +43,16 @@ void AmountsMenu::draw() {
...
+            l_healthNum = 1;  // Don't allow the player to go under 1 quarter heart
+        }
+     if(l_healthNum > l_hpNum) {
+            dComIfGs_setLife(l_hpNum); // Don't allow the player to go overflow        
+        }
+    dComIfGs_setLife(l_healthNum);
```
Comment (TakaRikka) [inline modules/menus/menu_amounts/src/amounts_menu.cpp]: this should follow consistent formatting with other cases

**Review by kipcode66 — APPROVED** [review]

**Comment by TakaRikka** [comment]
looks fine after clang check is fixed

**Pull request merged**